packages/amplify_analytics_pinpoint/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 0.1.4
 homepage: https://github.com/aws-amplify/amplify-flutter/tree/master/packages/amplify_analytics_pinpoint
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: '>=2.12.0 <3.0.0'
   flutter: ">=1.20.0"
```

Comment:
Copied from comment left on #485

Nit: This might be a good opportunity to rein in the consistency between " vs ' usage in our pubspec. I think the preferred convention is ' like you have it here for multi-part constraints and not even having quotes at all for single-part constraints i.e.:
```yaml
flutter: >=1.10.0
```

Reply:
changed to ^1.10.0